assets/src/edit-story/components/postLock/postLock.js

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import { useCallback, useEffect, useRef, useState } from 'react';
+import { useFeatures } from 'flagged';
+import { trackError } from '@web-stories-wp/tracking';
+
+/**
+ * Internal dependencies
+ */
+import { useAPI } from '../../app/api';
+import { useStory } from '../../app/story';
+import { useConfig } from '../../app/config';
+import PostLockDialog from './postLockDialog';
+
+function PostLock() {
+  const {
+    actions: { getStoryLockById, setStoryLockById },
+  } = useAPI();
+  const {
+    storyId,
+    userId,
+    allStoriesLink,
+    postLock: { interval: postLockInterval, enabled: postLockEnabled },
+  } = useConfig();
+
+  const { previewLink } = useStory(({ state: { story: { previewLink } } }) => ({
+    previewLink,
+  }));
+  const [showDialog, setShowDialog] = useState(false);
+  const [author, setAuthor] = useState({});
+  const { enablePostLocking } = useFeatures();
+
+  const closeDialog = useCallback(() => {
+    setShowDialog(false);
+    setStoryLockById(storyId);
+  }, [setStoryLockById, storyId]);
+
+  const doGetStoryLock = useCallback(() => {
+    if (enablePostLocking && postLockEnabled) {
```

Comment:
These variables have very similar names.

Let's replace `postLockEnabled` / `enabled` with `showPostLockingDialog` / `showDialog`

Reply:
Changed in https://github.com/google/web-stories-wp/pull/6497/commits/fcacf1e30d2c9e95b7700fc326651fa756cad34c